### PR TITLE
[PE] Add missing examples and properties; general cleanup

### LINKED ISSUE #811: [PE] Missing content
- [x] Add examples
- [x] Add `E(X + Y)  = E(X) + E(Y)` property
- [x] Add `Var(X + Y)` property
- [ ] Add Testes de Ajustamento do Qui-quadrado de Pearson
- [ ] Add regressao linear

### PR DESCRIPTION
**Pull request by randomicecube**
Partially closes #811

### CODE REVIEW

File: content/pe/0003-va-discretas.md [hunk 1]
```diff
@@ -389,7 +409,7 @@ Esta distribuição não faz parte da matéria lecionada no programa de 2021/22.
 
 :::
 
-:::tip[]
+:::tip[Motivação]
```
Comment (Pesteves2002) [inline content/pe/0003-va-discretas.md:412]: Muda o warning em cima para danger  para manter conformidade com o resto da distribuicoes q n foram lecionadas

File: content/pe/0009-teste-de-hipoteses.md [hunk 2]
```diff
@@ -322,6 +322,65 @@ Sendo assim, a região de rejeição é aproximadamente
...
+Devemos, ainda, definir inicialmente a região de rejeição de $H_0$: tratando-se
+de um teste unilateral superior (com $H_1: \sigma^2 > 4$), dizemos que a região
+de rejeição de $H_0$ é dada por $W = (c, +\infty)$.
+
+O valor observado, tendo em conta a estatística de teste, é dado por:
+
+$$
+\begin{aligned}
+t &= \frac{(n - 1)s^2}{\sigma_0^2} \\
+&= \frac{(10 - 1) \cdot 3.9(3)^{\smartcolor{orange}{*}}}{4} \\
```
Comment (Pesteves2002) [inline content/pe/0009-teste-de-hipoteses.md:356]: Falta saber onde os asterisco vai dar

**Review by Pesteves2002 — CHANGES_REQUESTED** [review]

**Comment by diogotcorreia** [comment]
Even though this isn't finished, I'm going to merge it as is. A future PR can be opened to finish the remaining tasks in #811.

**Review by diogotcorreia — APPROVED** [review]

File: content/pe/0002-variaveis-aleatorias.md [hunk 3]
```diff
@@ -235,6 +241,7 @@ As funções mais que relevantes que nos dão informações sobre VA's são:
   - $V(X) \geq 0$;
   - $V(X) = 0 \Leftrightarrow X$ constante;
   - $V(aX+b) = a^2V(x)$, para $a,b \in \R$.
+  - $V(X^2) = E([X^2]^2) - [E^2(X)]^2 = E(X^4) - E(X)^4$;
```
Comment (Pesteves2002) [inline content/pe/0002-variaveis-aleatorias.md:244]:
```suggestion
  - $V(aX+b) = a^2V(x)$, para $a,b \in \R$;
  - $V(X^2) = E([X^2]^2) - [E^2(X)]^2 = E(X^4) - E(X)^4$.
```

File: content/pe/0003-va-discretas.md [hunk 4]
```diff
@@ -13,6 +13,23 @@ type: content
...
+:::tip[Prova de Bernoulli]
+
+Damos o nome de **prova de Bernoulli** a qualquer experiência aleatória cujo espaço de resultados tem apenas dois eventos elementares: um evento a que damos o nome de **sucesso**, com probabilidade $p$, e um a que damos nome de **insucesso**, com probabilidade $1-p$.
+
+:::
+
+:::warning[Sucesso pode ser mau!]
+
+Enquanto que estamos habituados a associar sucesso a coisas boas e insucesso a coisas más, neste caso, o sucesso deve ser entendido apenas como **aquilo que queremos modelar**.  
+Sendo assim, por exemplo, se considerarmos a EA que verifica se o ecrã de um telemóvel se parte no primeiro ano de uso, o sucesso será "o ecrã partiu-se".
```
Comment (Pesteves2002) [inline content/pe/0003-va-discretas.md:27]:
```suggestion
Sendo assim, por exemplo, se considerarmos a Experiência $A$ que verifica se o ecrã de um telemóvel se parte no primeiro ano de uso, o sucesso será "o ecrã partiu-se".  
```

**Review by Pesteves2002 — CHANGES_REQUESTED** [review]

**Review by Pesteves2002 — APPROVED** [review]

**Pull request merged**